lib/streamlit/elements/widgets/time_widgets.py

```diff
@@ -556,39 +582,41 @@ def date_input(
     def _date_input(
         self,
         label: str,
-        value: DateValue = None,
+        value: DateValue | ellipsis = ...,
```

Comment:
Just curious the implication of ellipsis here

Reply:
The initial plan was to use the ellipsis object to indicate that the default value should be used since the `...` can be used as parameter input without requiring any import from the user. It would really have any other implications. But we eventually decided against doing this and instead changed it just to use literals. In this case: `"today"`.